internal/monitor/features/integration.feature

```diff
@@ -378,3 +378,39 @@ Feature: Integration Test
       | ""         | "2-2"       | "2-2" | "2-2" | "unity"    | "unity-iscsi" | "one-third" | "zero"  | "reboot" | 120      | 600        | 300     | 600           |
       | ""         | "1-1"       | "1-1" | "0-0" | "unity"    | "unity-nfs"   | "one-third" | "zero"  | "reboot" | 120      | 600        | 300     | 600           |
       | ""         | "2-2"       | "2-2" | "0-0" | "unity"    | "unity-nfs"   | "one-third" | "zero"  | "reboot" | 120      | 600        | 300     | 600           |
+
+  @powerflex-array-interface
+  Scenario Outline: Multi networked nodes with a failure against the array interface network
+    Given a kubernetes <kubeConfig>
+    And cluster is clean of test pods
+    And wait <nodeCleanSecs> to see there are no taints
+    And <podsPerNode> pods per node with <nVol> volumes and <nDev> devices using <driverType> and <storageClass> in <deploySecs>
+    Then validate that all pods are running within <deploySecs> seconds
+    When I fail <workers> worker nodes and <primary> primary nodes with <failure> failure for <failSecs> and I expect these taints <taints>
+    Then validate that all pods are running within <runSecs> seconds
+    And the taints for the failed nodes are removed within <nodeCleanSecs> seconds
+    Then finally cleanup everything
+
+    Examples:
+      | kubeConfig | podsPerNode | nVol  | nDev  | driverType | storageClass | workers     | primary | failure                | taints                             | failSecs | deploySecs | runSecs | nodeCleanSecs |
+      | ""         | "1-1"       | "1-1" | "1-1" | "vxflexos" | "vxflexos"   | "one-third" | "zero"  | "interfacedown:ens192" | "vxflexos.podmon.storage.dell.com" | 120      | 240        | 300     | 300           |
```

Comment:
I think putting the interface name in the Scenarios is a bit too limiting. Could we pass it in from the parameters like those you added in the Makefile? Especially if it's always the same interface. If it can be multiple interfaces, virtualize the names like interface-down:interface-a, reboot:interface-b so something similar. Then when running the test you can say interface-a=ens192.

Reply:
Sure, I can look into making this parameterized.

Reply:
Added parameterization of the interface names. In the feature file, the 'failure' field can take a parameter like before, but this time instead of a specific interface name it is the name of an environmental variable that will hold the name. The Makefile has reference to the env var. If not set, the test will return an error.